### PR TITLE
Update Primer

### PR DESCRIPTION
**Pull request by sophshep**
I've updated to our new version of Primer (`primer-css@7.0.0-2`), as listed here: https://github.com/primer/primer-css/pull/230#issuecomment-309608735. No issues! Everything worked as expected 🎉 

I'd love to merge this since it updates all the colors and other various improvements we've made to primer in the time since it was last included here!

cc/ @github/design-systems

### CODE REVIEW

File: node_modules/primer-states/build/build.css [hunk 1]
```diff
@@ -1 +0,0 @@
-.state{display:inline-block;padding:4px 8px;font-weight:600;line-height:20px;color:#fff;text-align:center;background-color:#999;border-radius:3px}.state-open,.state-proposed,.state-reopened{background-color:#6cc644}.state-merged{background-color:#6e5494}.state-closed{background-color:#bd2c00}.state-renamed{background-color:#fffa5d}
```
Comment (broccolini) [inline node_modules/primer-states/build/build.css]: If you need `primer-states` then include the `primer-labels` package. States were moved into the labels package, and the labels package was added to the product package. If you don't need states then you can delete all these files and references to them.

Comment (sophshep) [inline node_modules/primer-states/build/build.css]: 👍  ... i don't need them. will delete.

**Comment by nayafia** [comment]
@broccolini @sophshep hi! :wave: ok to merge this, or do we need to update version?

**Comment by sophshep** [comment]
> do we need to update version? 

@nayafia We should update to an even newer version! I'll take care of this this week.

**Comment by sophshep** [comment]
This is good to go from the design side. All markup has been updated to allow for new changes from the latest version of Primer.

One thing that has me stumped is the failing CI test. It says `$gray-300` is an undefined variable, but it's almost definitely not. [I tested it here](https://github.com/github/opensource.guide/pull/404/commits/5bd4dd95175ab8b951dcc70822bd06f6325dcf1f) and it compiled and changed visually, so I'm not sure how to troubleshoot. 

Any ideas @github/design-systems & @MikeMcQuaid?

**Comment by bkeepers** [comment]
@shawnbot, @sophshep: sorry for the confusion with tests failing. The fix was to `git add -f node_modules/primer*` (3c955ad) and then remove `node_modules` from `.gitignore` (longer explanation for why this was set up like this in the first place in 874a049).

Quora appears to be acting up, which was breaking the tests again, so I ignored that from the link checker.

**Review by WojasKrk — COMMENTED** [review]
Wojaskrk@gmail.com

File: node_modules/primer-states/build/build.css (hunk 1 above)
Comment (ZuzanaBittnerova) [reply node_modules/primer-states/build/build.css:1]: 5473cb94bc941f37e03eb4599db01ba82030508a

**Pull request merged**